### DISCUSSION TITLE
Is it possible to convert a development environment into a production one?

### DISCUSSION THREAD

**Discussion by carlosmartinezdelacasa** [ckan/ckan#7771] [General]
I wonder if there are manuals that indicate steps or references to follow to make this type of conversion possible.

As an example, in CKAN 2.9.x versions, if you install from source, there are components such as supervisor, uwsgi, nginx that are not installed in source (development environment) but if you want to pass this environment to production if they are necessary.

Is there documentation associated with these steps?

**Comment by kowh-ai** [comment]
How about [this doco](https://docs.ckan.org/en/2.9/maintaining/installing/deployment.html)
